src/main/Extensions/Base64Conversion/Base64Conversion.ts

```diff
@@ -0,0 +1,81 @@
+import type { AssetPathResolver } from "@Core/AssetPathResolver";
+import type { Extension } from "@Core/Extension";
+import type { Translator } from "@Core/Translator";
+import { type SearchResultItem, SearchResultItemActionUtility } from "@common/Core";
+import type { Image } from "@common/Core/Image";
+import type { InvocationArgument } from "@common/Extensions/Base64Conversion/InvocationArgument";
```

Comment:
You can simplify this import as it's exported in `common/Extensions/Base64Conversion/index.ts`:

```suggestion
import type { InvocationArgument } from "@common/Extensions/Base64Conversion";
```